stat.go

```diff
@@ -187,6 +187,9 @@ func parseStat(r io.Reader, fileName string) (Stat, error) {
 		err error
 	)
 
+	buf := make([]byte, 0, 8*1024)
```

Comment:
Maybe add a comment that we increased this due to long intr lines. Also might want to make this a const? But fine with me either way

Reply:
Added a comment.